editor/src/components/inspector/common/css-utils.spec.browser.ts

```diff
@@ -26,7 +27,7 @@ describe('toggle style prop', () => {
     await renderResult.dispatch(
       [
         toggleProperty(
-          TP.instancePath(TestScenePath, ['aaa', 'bbb']),
+          TP.appendNewElementPath(TestScenePath, ['aaa', 'bbb']) as InstancePath,
```

Comment:
TODO for next PR?

Reply:
Was done about 10 mins ago (for next PR) :D